adoc/chapters/programming_interface.adoc

```diff
@@ -1809,8 +1809,8 @@ info::device::max_work_item_sizes\<1>
 ----
 
     @ [.code]#id<1>#
-   a@ Returns the maximum number of work-items that are permitted of the
-      work-group of the [code]#nd_range#. The minimum value is latexmath:[(1)]
+   a@ Returns the maximum number of work-items that are permitted in a
+      work-group. The minimum value is latexmath:[(1)]
```

Comment:
Is the idea that this query is only meaningful for a 1-D parallel_for?  If so, should we say something like:

> Returns the maximum number of work-items that are permitted in a work-group for a kernel running in a one-dimensional index space.

Etc. for the other queries.

Reply:
I like that.  Applied your suggestion in 31e120f5.